apps/web/src/pages/tenants/components/list/TenantsList.tsx

```diff
@@ -1,3 +1,31 @@
-export function TenantsList() {
-  return null;
-}
+import { Container } from '@mantine/core';
+
+import { Table } from '../../../../design-system';
+import PageContainer from '../../../../components/layout/components/PageContainer';
+import { When } from '../../../../components/utils/When';
+import { columns } from './columns';
+import { useTenants } from '../../../../hooks/useTenants';
+import { Toolbar } from './ToolBar';
+
+export const TenantsList = ({ onAddTenantClick }: { onAddTenantClick: React.MouseEventHandler<HTMLButtonElement> }) => {
+  const { tenants, loading: tenantLoading } = useTenants();
+  const isLoading = tenantLoading;
```

Comment:
```suggestion
  const { tenants, loading: tenantsLoading } = useTenants();
```

unnecessary isLoading param